build-tools/packages/build-cli/src/commands/release/report-unreleased.ts

```diff
@@ -154,19 +153,39 @@ async function updateReportVersions(
 	version: string,
 	log: Logger,
 ): Promise<void> {
+	const clientPackage = "fluid-framework";
+
+	if (report[clientPackage] === undefined) {
```

Comment:
nit: not blocking - you use report[clientPackage] often enough that a local variable declared right after clientPackage would be helpful.